src/collect/collector.rs

```diff
@@ -184,7 +184,14 @@ impl Collectors {
             .downcast_ref::<Collect>()
             .ok_or_else(|| anyhow!("wrong subcommand"))?;
 
-        if collect.args()?.stack {
+        if collect.args()?.probe_stack
+            && collect.args()?.packet_filter.is_none()
+            && collect.args()?.meta_filter.is_none()
+        {
+            bail!("Auto-probe mode requires filtering (`--filter-packet` or `--filter-meta`)");
```

Comment:
at least one of, right?

Reply:
Yes, at least one. Am I missing something? :)

Reply:
no :) 
I meant as it is phrased (the part inside the parenthesis) it may seem one or the other while both are usable simultaneously.

Reply:
IMO as this is the error message, that's fine. One filter is missing. However those options are not described in the `--probe-stack` help. (And it seems the error message above still refers to "auto-probe"). I'll fix that.

Reply:
thanks!